### DISCUSSION TITLE
Updated Roadmap?

### DISCUSSION THREAD

**Discussion by lylerolleman** [AvaloniaUI/Avalonia#12932] [Q&A] [answered]
Does anyone know if there is an updated roadmap somewhere?

There is a link to an issue in the README, but it was closed (looks like around when v11 was released)

In particular I'm interested in features like out of the box navigation support (including in the UI, ie a Shell/NavigationView), cross platform property storage/secure storage as well as accessibility support (according to the v11 release it currently supports windows and macOS with other platforms coming later... but when? Government legislation in many jurisdictions makes this somewhat urgent)

**Comment by MikeCodesDotNET** [comment, marked as answer]
We have an internal roadmap guiding our efforts, but it's not yet in a publishable state. Most of the work for v12 is still under discussion, so we can't provide concrete details now. 

However, I'd like to point you to our [Avalonia.Labs project](https://github.com/AvaloniaUI/Avalonia.Labs), which serves as the home for experimental UI controls. It already includes a navigation control that you might find helpful.

**Reply by lylerolleman** [reply]
I don't see a general navigation control in Avalonia Labs

I know there are some things in Awesome Avalonia, or just rolling our own which isn't that big of a deal, but it is a bit of a barrier to entry. 

I'll keep my fingers crossed then that at least accessibility makes it onto your roadmap (that's one I'd really rather not do myself...)

**Reply by MikeCodesDotNET** [reply]
Take a look at the [Navigation control](https://github.com/AvaloniaUI/Avalonia.Labs/tree/main/src/Avalonia.Labs.Controls/Navigation). Creating a universal navigation control that excels across all platforms and use cases is a complex task.

Regarding accessibility, it's unquestionably slated for future development. We aim to address it before the release of version 12, but it all comes down to resource allocation.

**Reply by lylerolleman** [reply]
Thanks for the link (not sure how I/github search missed that...)

And glad to hear accessibility is a priority

**Reply by yowl** [reply]
Accessibility for Automation Testing would be very useful

**Reply by maxkatz6** [reply]
@yowl accessibility (including for automation testing) is already available in stable versions for Windows and macOS. 
Any other platform would require custom implementation.

**Reply by yowl** [reply]
I should have mentioned Wasm was where I would like it ideally, although desktop would be ok in the short term.
